public/index.html

```diff
@@ -7,5 +7,672 @@
   </head>
   <body>
     <div id="root"></div>
```

Comment:
this can be removed if you don't use react

Reply:
For now, I left it, cause when I remove it there are some errors related to React. But I understood that it is redundant unless I use React.